### ISSUE TITLE
Roblox giving me error code 279. ID = 17 when joining a new server with my 5 alts, although it worked before.

### ISSUE THREAD

**Issue by SebTheDev1** [ic3w0lf22/Roblox-Account-Manager#374]
I was playing a game with 5 alt accounts and teleported all of them to a new server(and it worked). When I joined back the main lobby and retried teleporting back, most of the alt accounts showed the message in the title. This may be due to Roblox’s Rate Limiting as stated in the FAQ, but I’m still not sure.

**Comment by Lunaisdedd** [comment]
Yes, i've had this problem its the rate limit. Very frustrating but nothing u can do about it.
